examples/consul-ami/README.md

```diff
@@ -72,6 +72,7 @@ Your code should look more like this:
   "provisioners": [{
     "type": "shell",
     "inline": [
+      "sudo yum install -y git || true",
```

Comment:
Ah, good fix. I'm not a fan of the `|| true`, as Ubuntu users will get a confusing error message.

How about just putting a `NOTE` just below this sample code that says Amazon Linux users will need to install Git?